### PR TITLE
Release/v1.1.6

### PR DESCRIPTION
**Pull request by kevinkirkup**

### CODE REVIEW

**Review by Try75 — DISMISSED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
 dist: xenial
-sudo: required
 services:
   - docker
 language: go
 go:
   - "1.13.x"
```
Comment (Try75) [inline .travis.yml]: Does this need to be 1.14.x?
Also, does a docker image need to be updated?

**Review by Try75 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,17 @@
 ## 1.1.6 (Unreleased)
```
Comment (Try75) [inline CHANGELOG.md:1]: Does the release date need to be entered or does that happen later?

Comment (kevinkirkup) [reply CHANGELOG.md:1]: That  gets updated by the Terraform team during their release process.

**Comment by kevinkirkup** [comment]
Closing. Need to update to remove Terraform Repo. ugh.

**Pull request closed without merging**